wire-ios/Wire-iOS/Sources/Helpers/syncengine/ZiphySession.swift

```diff
@@ -30,19 +37,31 @@ extension ZMUserSession: ZiphyURLRequester {
             preconditionFailure("request does not contain a valid URL")
         }
 
-        return doRequest(withPath: requestPath,
-                         method: .methodGET,
-                         type: .giphy,
-                         completionHandler: completionHandler)
+        return doRequest(
+            withPath: requestPath,
+            method: .methodGET,
+            type: .giphy,
+            completionHandler: completionHandler
+        )
     }
 
     public func cancelZiphyRequest(withRequestIdentifier requestIdentifier: ZiphyRequestIdentifier) {
         guard let requestIdentifier = requestIdentifier as? ProxyRequest else { return }
-        cancelProxiedRequest(requestIdentifier)
+        userSession.cancelProxiedRequest(requestIdentifier)
     }
 
-    private func doRequest(withPath path: String, method: ZMTransportRequestMethod, type: ProxiedRequestType, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void) -> ProxyRequest {
-        return proxiedRequest(path: path, method: method, type: type, callback: completionHandler)
+    private func doRequest(
+        withPath path: String,
+        method: ZMTransportRequestMethod,
+        type: ProxiedRequestType,
+        completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void
+    ) -> ProxyRequest {
+        return userSession.proxiedRequest(
```

Comment:
```suggestion
        userSession.proxiedRequest(
```
Return can be omitted here, not sure if we want it?